Update Wall of Shame (#27)
BlackWolfWoof/yagpdb-cc: README.md

## 💩Wall of shame💩

> <sub> I hate to see when people just remove the copyright from code and think that is perfectly fine. I have created this GitHub repository because I wanted to share my cool ideas with the community. Going ahead and ripping someone's name from their code, or even replacing it with their own is unaccaptable. For that reason, I created this wall of shame and if you find yourself here you should know that you are the reason I hate to publish my code. And that I would share more stuff if you wouldn't have been such an asshole.

Reviewer: ```suggestion
<sub> I hate to see when people just remove the copyright from code and think that is perfectly fine. I have created this GitHub repository because I wanted to share my ideas with the community.
Going ahead and ripping someone's name from their code, or even replacing it with their own is unacceptable. 
For specifically that reason, I created this wall of shame; should you find yourself here, know that you are the reason why I hate to publish my projects and that I would share more stuff, if it wasn't for you being such a cheapskate.
```

Author: Yeah much better
